### PR TITLE
Adding a note about TP PSA restricted enforcement

### PR DESCRIPTION
**Pull request by bergerhoffer**
Version(s):
4.13

Issue:
None - per @mburke5678's request

Link to docs preview:
https://59498--docspreview.netlify.app/openshift-enterprise/latest/authentication/understanding-and-managing-pod-security-admission.html

QE review:
- [ ] QE has approved this change.

Additional information:
4.12 will be done separately since there is an additional item to note

### CODE REVIEW

**Comment by bergerhoffer** [comment]
@gkarager @stlaz PTAL. It was requested to make mention of the TP feature set option on the actual PSA page too.

Preview: https://59498--docspreview.netlify.app/openshift-enterprise/latest/authentication/understanding-and-managing-pod-security-admission.html

**Comment by mburke5678** [comment]
@bergerhoffer One tiny nit. Otherwise LGTM

File: modules/security-context-constraints-psa-synchronization.adoc [hunk 1]
```diff
@@ -9,11 +9,18 @@
 
 {product-title} includes link:https://kubernetes.io/docs/concepts/security/pod-security-admission[Kubernetes pod security admission]. Globally, the `privileged` profile is enforced, and the `restricted` profile is used for warnings and audits.
 
+[NOTE]
+====
+If you want to preview _enforcing_ the `restricted` profile on your non-production cluster as a Technology Preview, you can enable the `TechPreviewNoUpgrade` feature set. If you enable the `TechPreviewNoUpgrade` feature set, pods are rejected if they violate pod security standards, instead of only logging a warning.
```
[the remark this answers is not in the document]
Comment (bergerhoffer) [reply modules/security-context-constraints-psa-synchronization.adoc]: Updated per other PR

**Comment by bergerhoffer** [comment]
@gkarager Can you PTAL? This is the same as the 4.12 PR except without the part of only being activated after cluster install.

https://59498--docspreview.netlify.app/openshift-enterprise/latest/authentication/understanding-and-managing-pod-security-admission.html

**Comment by kalexand-rh** [comment]
The `branch/enterprise-4.14` label has been added to this PR.

This is because your PR targets the `main` branch and is labeled for enterprise-4.13. And any PR going into `main` must also target the latest version branch (enterprise-4.14).

If the update in your PR does NOT apply to version 4.14 onward, please re-target this PR to go directly into the appropriate version branch or branches (enterprise-4.x) instead of main.

**Review by jldohmann — APPROVED** [review]
LGTM

**Comment by kalexand-rh** [comment]
> @gkarager Can you PTAL? This is the same as the 4.12 PR except without the part of only being activated after cluster install.
> 
> https://59498--docspreview.netlify.app/openshift-enterprise/latest/authentication/understanding-and-managing-pod-security-admission.html

@gkarager, will you PTAL?

**Comment by gkarager** [comment]
> > @gkarager Can you PTAL? This is the same as the 4.12 PR except without the part of only being activated after cluster install.
> > https://59498--docspreview.netlify.app/openshift-enterprise/latest/authentication/understanding-and-managing-pod-security-admission.html
> 
> @gkarager, will you PTAL?

@kalexand-rh this link says "Site Not Found
Looks like you've followed a broken link or entered a URL that doesn't exist on Netlify."

**Pull request closed without merging**